### DISCUSSION TITLE
About the loss on the progress bar

### DISCUSSION THREAD

**Discussion by Struggle-Forever** [Lightning-AI/pytorch-lightning#13842] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
![image](https://user-images.githubusercontent.com/29114869/180907549-d7619959-d6c8-4cd3-8417-04ab91dacccc.png)

The loss in the red box indicates? 
My understanding: loss_step denotes the loss of the current step (batch-level), loss_epoch denotes the loss of an epoch. So, the loss in the red box denotes ?

**Comment by rohitgr7** [comment, marked as answer]
ideally it is just the step loss, but you might notice that it's different from `loss_step`. The reason is `loss` is a running loss with certain window.

There is a [tracking issue](https://github.com/Lightning-AI/lightning/issues/9372), that will remove the averaging window.

**Reply by Struggle-Forever** [reply]
Thank you very much!
